pkg/controllers/clusterset/clustersetmapper/clusterset_controller.go

```diff
@@ -221,15 +217,23 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	//add clusterset clusterrole
 	adminroleName := utils.GenerateClustersetClusterroleName(clusterset.Name, "admin")
-	adminRole := buildAdminRole(clusterset.Name, adminroleName)
+	adminRole := clustersetutils.BuildAdminRole(clusterset.Name, adminroleName)
```

Comment:
I think you should put line 219 in the buildxxx func.

Reply:
done